### PR TITLE
Add valid examples for migrating investors

### PR DESCRIPTION
**Pull request by hoto**
Added two valid examples to `POST /migrate`, one for `INDIVIDUAL` and one for `CORPORATE`.
Existing example was terrible and confusing.

BEFORE:

Screenshot 2020-06-16 at 20 03 05

AFTER:

image

### CODE REVIEW

File: source/includes/_investors.md [hunk 1]
```diff
@@ -1997,7 +1997,7 @@ Lists all transfers out for all investors.
 
 If you are migrating your existing investors over to Goji, then you need to onboard the existing investors using the dedicated [investor migration endpoint](#investors-post-platformapi-migrate) and include `migrationDetails` data.
 
-The `migrationDetails` allows the platform to provide an existing investor ID where applicable and provide the dilligence status as agreed with Goji before migration takes place. 
+The `migrationDetails` allows the platform to provide an existing investor ID where applicable and provide the diligence status as agreed with Goji before migration takes place.
```
Comment (hoto) [inline source/includes/_investors.md:2000]: I don't understand this.
I've copied this from the description above but why does it say `existing investor ID where applicable` if it's mandatory?

File: source/includes/_investors.md [hunk 2]
```diff
@@ -2015,97 +2015,149 @@ Any existing ISA data is preserved.
...
+    "telephoneNumber": "07123456789"
+  }
 }
 ```
 ### Description
 Migrates an existing investor onto the Goji system.
+
+<aside class="notice">
+The required <code>migrationDetails</code> allows the platform to provide an existing investor client ID where applicable and provide the diligence status as agreed with Goji before migration takes place. 
+</aside>
```
Comment (hoto) [inline source/includes/_investors.md:2159]: I don't understand this.
I've copied this from the description above but why does it say `existing investor ID where applicable` if it's mandatory?

**Comment by deanrecord** [comment]
I think you can use the /migrate endpoint and it will actually create an investor without specifying that migration details. Now I'm not sure what the difference between a migrate or creation is internally in the investor app. Another question for Dave I think.

File: source/includes/_investors.md (hunk 2 above)
Comment (hoto) [reply source/includes/_investors.md:2159]: image

image

But still the  `migrationDetails.existingClientId` can be empty.
This is so weird.
image

**Comment by hoto** [comment]
> I think you can use the /migrate endpoint and it will actually create an investor without specifying that migration details. Now I'm not sure what the difference between a migrate or creation is internally in the investor app. Another question for Dave I think.

That's answered here https://goji-investments.slack.com/archives/C5225EHU7/p1592393432022600

The `migrationDetails.existingClientId` can be either "our" existing client id or the platform "investor" id (whatever they have in their system).
This acts in two ways - either migrate our investor or create a brand new investor but keep track of the platforms original investor id but don't actually use it.

**Pull request merged**